src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -661,4 +668,11 @@ public ResponseEntity getAsyncJobResult(@RequestHeader(CURRENT_USER_KEY) String
                 .ok()
                 .body(asyncJobsManager.getJobResult(currentUser, jobId));
     }
+    @GetMapping(value = "/announcements/active")
+    public ResponseEntity<List<String>> activeAnnouncements() {
```

Comment:
Make sure this endpoint returns ResponseEntity<Announcements>

Reply:
Oops, I typed the wrong thing. I meant to say return ResponseEntity<Annoucnements>